### PR TITLE
chore: tweak biome config

### PR DESCRIPTION
**Pull request by aslilac**
some config files in site/ have been getting missed by the current biome config. this addresses that (and formats them).

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -642,7 +642,6 @@ vpn/vpn.pb.go: vpn/vpn.proto
 site/src/api/typesGenerated.ts: $(wildcard scripts/apitypings/*) $(shell find ./codersdk $(FIND_EXCLUSIONS) -type f -name '*.go')
 	# -C sets the directory for the go run command
 	go run -C ./scripts/apitypings main.go > $@
-	(cd ./site && npx biome format --write ./src/api/typesGenerated.ts)
 	./scripts/pnpm_install.sh
```
Comment (matifali) [inline Makefile:645]: This line is redundant now as we are not running any `pnpm` script after this.

File: site/biome.jsonc [hunk 2]
```diff
@@ -44,5 +47,6 @@
 				}
 			}
 		}
-	}
+	},
+	"$schema": "https://biomejs.dev/schemas/1.9.4/schema.json"
```
Comment (matifali) [inline site/biome.jsonc:51]: Would this need to be manually bumped when @dependabot bumps biome version?

**Review by matifali — APPROVED** [review]

File: site/biome.jsonc (hunk 2 above)
Comment (BrunoQuaresma) [reply site/biome.jsonc:51]: I think so.

**Review by BrunoQuaresma — APPROVED** [review]

**Review by jaaydenh — APPROVED** [review]
LGTM

File: site/biome.jsonc (hunk 2 above)
Comment (aslilac) [reply site/biome.jsonc:51]: unfortunately yeah, but it's worth having for the autocompletion help

**Pull request merged**